pkg/plugin/v2/scaffolds/internal/templates/controller/controller.go

```diff
@@ -63,7 +67,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	{{ if .WireResource }}
```

Comment:
You are adding an extra blank line here. It may get removed later by the go formatter but it is better if we don't have it.
```suggestion
	{{ if .WireResource -}}
```